plugin-server/src/worker/ingestion/ingest-event.ts

```diff
@@ -25,6 +25,8 @@ export async function ingestEvent(hub: Hub, event: PluginEvent): Promise<IngestE
             uuid! // it will throw if it's undefined
         )
 
+        await hub.eventsProcessor.dispatchEventToRunner(event)
```

Comment:
Looks like this is the event before we run `processEvent` on it (e.g. before we apply geoIp etc), but `onEvent` etc should get the event with geoIp and other plugins applied, right? So maybe refactor this so that we get the processedEvent and actions here to dispatch to runner.

Reply:
That's not the case, but I see why you think it is - we've had a terrible naming conflict issue in the plugin server for a while.

We have:

`processEvent` which relates to plugins running on an event and `processEvent` which is doing elements and persons stuff when ingesting the event.

In this case the main thread has already:

1. Ran the plugins `processEvent`
2. Got the result back and passed it to this function

Thus, we're sending the event to the `onEvent` queue before all the persons stuff. Why? For now, just to keep this the same as it used to be. Remember how the main thread currently runs `ingestEvent` and `onEvent` concurrently.

This is something we may want to change (e.g. pass the event with person props) but for now this is backwards compatible and doesn't change the way `onEvent` behaves